elements/pfe-tabs/README.md

```diff
@@ -39,6 +39,12 @@ Add the content for your tab panel here.
 
 ## Attributes
 
+**`pfe-variant="wind"`** (observed)
```

Comment:
@starryeyez024 You and @kylebuch8 opened identical documentation updates: https://github.com/patternfly/patternfly-elements/pull/824. I left my notes on his - the same applies to this one though.